Ports/ffmpeg/package.sh

```diff
@@ -1,20 +1,22 @@
 #!/usr/bin/env -S bash ../.port_include.sh
 port=ffmpeg
-version=4.4
+version=5.0
 useconfigure=true
-depends=("libiconv" "libtiff" "xz" "bzip2")
-files="https://ffmpeg.org/releases/ffmpeg-${version}.tar.gz ffmpeg-${version}.tar.gz a4abede145de22eaf233baa1726e38e137f5698d9edd61b5763cd02b883f3c7c"
+depends=("libiconv" "libtiff" "xz" "bzip2" "SDL2")
+files="https://ffmpeg.org/releases/ffmpeg-${version}.tar.gz ffmpeg-${version}.tar.gz 7bf52bc242b5db8df67c62cb826df134d917dedcf6abf1289e15e4057bcc1750"
 auth_type="sha256"
 installopts=("INSTALL_TOP=${SERENITY_INSTALL_ROOT}/usr/local")
 configopts=("SRC_PATH=.")
 
 configure() {
     run ./configure \
-        --arch="x86_32" \
+        --target-os=none \
+        --arch="${SERENITY_ARCH}" \
```

Comment:
Does this work even though our 32-bit arch is `i686` and not `x86_32` as the configure script received before?